.github/workflows/release.yml

```diff
@@ -60,9 +62,16 @@ jobs:
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
         with:
           name: tmr-debs-${{ env.ARTIFACT_SUFFIX }}
-          path: tmr-debs-${{ env.ARTIFACT_SUFFIX }}.tar.gz
+          path: "target/debian/*.deb"
```

Comment:
Do those files have unique names? I see above that they are for some reason compressed into a tar.gz, but why so?

I'd rename the deb into something with `ARTIFACT_SUFFIX`, avoid compressing them, and upload the renamed .deb file. What do you think, @veluca93?